dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_Analysis_test_group_2_fitting.rst

```diff
@@ -1,132 +1,225 @@
-- Dialog should appear. In the runs box **of the new dialog that appears**, type "20918-20" and hit Start.
-- Sequential fit of runs 20918, 20919, 20920 should happen one after the other.
 
+**Time required 5 minutes**
 
+- Following the set up from :ref:`test_1`, load runs ``20918-20`` and reset 
+  ``Time Start = 0.113`` and ``Time End = 15``
+- Go to the **Sequential Fitting** tab
+	- Click **Sequentially fit all**
 
-Test 3: co-added fit
+-----------
+	
+.. _test_3:
+	
+Test 3: Co-added Fit
 --------------------
-- On the *Home* tab, load EMU 20918-20.
-- Go to *Data Analysis* tab. Stale errors should be cleared from the function browser.
-- In the drop-down, there should only be one workspace (``EMU00020918-20; Pair; long; Asym; #1``).
-- Fit as before. Graph should be updated.
-
-Test 4: simultaneous fit across runs
--------------------------------------
-- With the same runs (20918-20) selected, select the "Simultaneous" radio button option.
-- Drop-down list should have three workspaces in it now, for the three runs that will be fitted.
-- The "Label" box should be enabled. By default, this will contain the run number string ("20918-20") - add "SimFit" to the end of the string.
-- Keep the same fit function, but use the "Global" checkboxes to mark ``A``, ``Omega``, ``Phi`` and ``Sigma`` as global.
-- Fit the data. Note that plot will *not* be updated at present.
-  (If you want to plot results, see the ``MuonSimulFit_20918-20SimFit`` workspace group).
-- Use the ``<<`` and ``>>`` buttons, or drop-down list, to see the fitted parameters for each run in the function browser.
-
-Test 5: simultaneous fit across groups
+
+**Time required 5 minutes**
+
+- Following :ref:`test_2`, in the *Loading* section, check **Co-Add** to load 
+  ``20918-20`` as one run
+- Go to **Fitting** tab
+	- In the **Select Workspace** drop-down, there should only be one workspace
+	  ; ``EMU00020918-20; Pair Asym; long; MA``
+	- Click **Fit** as before. The plot should be updated
+	
+.. image:: /images/MuonAnalysisTests/fitting_test3.png
+	:align: center
+	:alt: fitting_test3.png
+
+-----------
+
+.. _test_4:
+	
+Test 4: Simultaneous Fit Across Runs
+------------------------------------
+
+**Time required 5 minutes**
+
+- Following :ref:`test_3`, uncheck **Co-Add**
+- Go to the **Fitting** tab
+	- Check the **Simultaneous fit over** checkbox, and change from **Run** 
+	  to **Group/Pair**
+	- The **Display parameters for** drop down should contain three workspaces
+	- Using the same fit function as before, check the **Global** checkbox for 
+	  the parameters **A**, **Omega**, **Phi** and **Sigma**
+	- Click **Fit**
+	- Use the **<<** and **>>** buttons, or drop-down list, to see the fitted 
+	  parameters for each run in the function browser.
+	- The fit should look something like this:
+	
+.. image:: /images/MuonAnalysisTests/fitting_test4.png
+	:align: center
+	:alt: fitting_test4.png
```

Comment:
this would be better as a tile plot and we should tell the users to create a tile plot too by selecting it in the plotting window